src/oatpp/web/server/HttpProcessor.cpp

```diff
@@ -125,19 +125,19 @@ bool HttpProcessor::processNextRequest(ProcessingResources& resources) {
 
   } catch (oatpp::web::protocol::http::HttpError& error) {
 
-    auto response = resources.components->errorHandler->handleError(error.getInfo().status, error.getMessage(), error.getHeaders());
-    response->send(resources.connection.get(), &resources.headersOutBuffer, nullptr);
+    auto response1 = resources.components->errorHandler->handleError(error.getInfo().status, error.getMessage(), error.getHeaders());
```

Comment:
Maybe, let's just reuse the `response` variable declared before the `try` block.
just to avoid the `..1` vars.

Reply:
I thought that may change `shared_ptr` destruction order - if we reuse the variable, the old `shared_ptr` may be destructed inside the catch blocks. Is this fine?